common/src/python/centers/center_group.py

```diff
@@ -709,3 +709,29 @@ class REDCapProjectInput(BaseModel):
     study_id: str
     project_label: str
     projects: List[REDCapFormProject]
+
+
+class REDCapModule(BaseModel):
+    """Information required to create a REDCap project for a module."""
+    model_config = ConfigDict(populate_by_name=True,
+                              alias_generator=AliasGenerator(alias=kebab_case))
+    label: str
+    title: str
+    template: Optional[str] = None
```

Comment:
This is a reference of some sort, correct? Maybe add some detail to the class document block.

Reply:
It is the XML template name (if different from module label), updated the docstring.